### PR TITLE
[Task 1848099]add to string for common models and add list webapp run…

### PR DESCRIPTION
**Pull request by andxu**
…time functionality

### CODE REVIEW

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/AzureAppService.java [hunk 1]
```diff
@@ -104,6 +108,11 @@ public IWebApp webapp(WebAppEntity webAppEntity) {
                 .collect(Collectors.toList());
     }
 
+    public @Nonnull List<Runtime> listWebAppRuntimes(@Nonnull OperatingSystem os, @Nonnull JavaVersion version) {
+        return Runtime.values().stream().filter(r -> Objects.equals(os, r.getOperatingSystem()) && Objects.equals(version, r.getJavaVersion()))
```
Comment (Flanker32) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/AzureAppService.java:112]: `Runtime` includes runtime for web app and functions, we may need to add correspond constant/static method for each service

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/WebContainer.java [hunk 2]
```diff
@@ -82,4 +83,16 @@ public boolean equals(Object target) {
     public int hashCode() {
         return Objects.hash(value);
     }
+
+    @Override
+    public String toString() {
+        if (this.equals(JAVA_OFF)) {
```
Comment (Flanker32) [inline azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/WebContainer.java:89]: do we need to handle `JAVA_OFF` here? If so, do we also need to handle `JAVA_OFF` in `JavaVersion`

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/AzureAppService.java (hunk 1 above)
Comment (andxu) [reply azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/AzureAppService.java:112]: this PR is only for webapp

File: azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/WebContainer.java (hunk 2 above)
Comment (Flanker32) [reply azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/WebContainer.java:90]: May modify value in constructor and return value directly in `toString`

**Review by Flanker32 — APPROVED** [review]
todo:

Refactor enum with name/label interface

**Review by Flanker32 — DISMISSED** [review]

**Review by Flanker32 — APPROVED** [review]

**Pull request merged**